documentation/en/cli-lotus-miner.md

```diff
@@ -598,7 +598,6 @@ USAGE:
 COMMANDS:
    status                Get the seal status of a sector by its number
    list                  List sectors
-   refs                  List References to sectors
```

Comment:
this should get restored with docsgen-cli after the `sectors refs` cmd restoration

Reply:
You are right, fixed this.